source/server/http/admin.cc

```diff
@@ -539,9 +539,10 @@ Http::Code AdminImpl::handlerConfigDump(absl::string_view, Http::HeaderMap& resp
     auto& any_message = *(dump.add_configs());
     any_message.PackFrom(*message);
   }
+  const auto redacted = MessageUtil::redact(dump);
```

Comment:
Tests?

Reply:
Yeah, still to do. I wanted to get this PR in front of everybody first to make sure I'm going the right direction.